classes/Task.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private int $customer_id;
+    private ?int $executor_id;
+    private string $current_status = self::STATUS_NEW;
+
+    const STATUS_NEW = 'status_new';
+    const STATUS_CANCELED = 'status_canceled';
+    const STATUS_IN_WORK = 'status_in_work';
+    const STATUS_PERFORMED = 'status_performed';
+    const STATUS_FAILED = 'status_failed';
+    const ACTION_CANCEL = 'action_cancel';
+    const ACTION_RESPOND = 'action_respond';
+    const ACTION_EXECUTE = 'action_execute';
+    const ACTION_REFUSE = 'action_refuse';
+
+    public function __construct(int $customer_id, int $executor_id = null)
+    {
+        $this->executor_id = $executor_id;
+        $this->customer_id = $customer_id;
+    }
+
+    // Объект класса создаёт заказчик, исполнитель, скорее всего, подключится позже
+    // Функция установки id исполнителя задания
+    public function setExecutorId(int $executor_id)
+    {
+        $this->executor_id = $executor_id;
+    }
```

Comment:
В методах, которые ничего не возвращают, лучше указывать `void` как результат.
В методах-сеттерах (которые устанавливают какие-то значения свойств), лучше возвращать ссылку на класс, чтобы можно было вызывать методы цепочкой.
Либо так (без цепочного вызова):
```suggestion
    public function setExecutorId(int $executorId): void
    {
        $this->executorId = $executorId;
    }
```
Либо так:
```suggestion
    public function setExecutorId(int $executorId): self
    {
        $this->executorId = $executorId;
        
        return $this;
    }
```
Чтобы можно было делать так:
```php
$task = new Task($customerId);
$task->setExecutorId($executorId)
    ->setCurrentStatus(...)
    ->anotherMethod(...);
```